### DISCUSSION TITLE
Template Multi-App Workspace with all the various json's

### DISCUSSION THREAD

**Discussion by JeremyVyska** [microsoft/alguidelines#232] [General]
What would be great is a template of a MAA workspace with all the various application best practices.  This should include workspace settings, launch.json examples, app.json examples, etc.

@alphakintari Maybe another you and I collab thing?

**Comment by AlphaKintari** [comment]
Makes sense.  There are more then a few options that are not obvious if you are creating any of those files from scratch.  Also might be good to point out how VSCode selects which file to use if there are multiple declarations.
IE.  Settings.json file in local .vscode folder vs Settings in the .workspace.
